pkg/ccl/backupccl/restoration_data.go

```diff
@@ -63,7 +64,8 @@ type restorationDataBase struct {
 	// spans is the spans included in this bundle.
 	spans []roachpb.Span
 	// rekeys maps old table IDs to their new table descriptor.
-	rekeys []execinfrapb.TableRekey
+	rekeys       []execinfrapb.TableRekey
```

Comment:
nit: rename to `tableRekeys`, and a comment on `tenantRekeys`

Reply:
Done, but isn't "TableRekey" a misnomer ? this should be DescRekey, right?